homeassistant/components/camera/webrtc.py

```diff
@@ -333,7 +333,7 @@ async def ws_get_client_config(
         vol.Required("type"): "camera/webrtc/candidate",
         vol.Required("entity_id"): cv.entity_id,
         vol.Required("session_id"): str,
-        vol.Required("candidate"): str,
+        vol.Required("candidate"): dict,
```

Comment:
Can we validate the dict?

Probably the best would be to write a custom validator/parser, which returns directly the RTCIceCandidateInit. So we can avoid the code below

Reply:
That's now done